src/search/algorithms/segmented_vector.h

```diff
@@ -182,7 +182,7 @@ class SegmentedArrayVector {
         : elements_per_array((assert(elements_per_array_ > 0),
                               elements_per_array_)),
           arrays_per_segment(
-              std::max(SEGMENT_BYTES / (elements_per_array * sizeof(Element)), size_t(1))),
+              std::max(SEGMENT_BYTES / (elements_per_array * sizeof(Element)), static_cast<size_t>(1))),
```

Comment:
This looks better, but is it something the style checks should do?

Reply:
Uncrustify produces `size_t (1)`, note the space. Do you prefer this? I'm indifferent.

Reply:
That is worse, but the question remains in both cases: should a style checker introduce casts?

Reply:
Would "1U" a better option that doesn't cause warnings or style-checker complaints here?

Reply:
Yes, let's use "1U" here. To clarify, **I** introduced the static_cast, not uncrustify.

Reply:
`1U` sounds good. What did uncrustify suggest here (or did you just stumble over this dynamic cast and wanted to clean it up)?

Reply:
Jendrik wrote that uncrustify wanted to insert a space: `size_t (1)`.